behave/features/steps/fala_end_to_end_tests.py

```diff
@@ -68,18 +77,23 @@ def step_impl_apply_filter(context):
     'the result page containing "{location}" is updated to apply the filter "{filter_label}"'
 )
 def step_impl_update_result_page(context, location, filter_label):
-    current_path = context.helperfunc.get_current_path()
-    title_xpath = context.helperfunc.find_by_xpath("//html/body/div/main/div/div/h1")
+    current_url = context.helperfunc.driver().current_url
+    title_xpath = context.helperfunc.find_by_xpath(
+        "//html/body/div/main/div/div/h1"
+    ).text.replace("\n", " ")
     result_container_xpath = context.helperfunc.find_by_xpath(
         '//div[@class="search-results-container"]'
     )
     updated_result_number_paragraph = context.helperfunc.find_by_xpath(
         '//p[@class="govuk-body"]'
     )
+    location = location.replace(" ", "+")
```

Comment:
Naming a variable the same as a parameter is confusing. Can rename the variable please.

Reply:
renamed here and in the other similar steps